### ISSUE TITLE
:bug: Discover hub not working after the recent update (could also be SkinVariables or TMDb Helper issue)

### ISSUE THREAD

**Issue by OfficerKD637** [jurialmunkey/skin.arctic.fuse#899]
### Skin section

Widgets

### Current Behavior

Hi Jurial!

After the recent update to all the 3 (Fuse, SV and TMDb Helper) the Discover hub is no longer working.

Here is the Skin Variables error:

``` 
error <general>: [xkb] couldn't find a Compose file for locale "C" (mapped to “C”)
error <general>: [xkb] couldn't find a Compose file for locale "C" (mapped to “C”)
error <general>: [xkb] couldn't find a Compose file for locale "C" (mapped to “C”)

error <general>: Misplaced +
error <general>: Error parsing boolean expression + skin.hassetting(indicator.ratingscircle)
error <general>: Misplaced +
error <general>: Error parsing boolean expression + skin.hassetting(indicator.progresspercent)
error <general>: Misplaced +
error <general>: Error parsing boolean expression + skin.hassetting(indicator.episodetypebanner)

error <general>: GetDirectory - Error getting -

error <general>: EXCEPTION Thrown (PythonToCppException) : ->Python callback/script returned the following error <-
Error Type: <class'AttributeError'>
Error Contents: 'NoneType' object has no attribute 'copy' 
AttributeError: 'NoneType' object has no attribute 'copy'

->End of Python script error report<-


error <general>: GetDirectory - Error getting
plugin://script.skinvariables/?info=get_filter_dir&library=video&sort_by=popularity&sort_how=desc&&plugin://plugin.video.themoviedb.helper/?info=discover&with_separator=AND&with_id=True&nextpage=false&fanarttv=false&cacheonly=true&length=1&tmdb_type=movie&length=2&sort_by=popularity.desc
```

No idea why this could be happening and wasn’t sure if it’d be more appropriate to post this bug under Skin Variables.

### Expected Behavior

Discover hub working again.

### Steps To Reproduce

1. Go to the Discover hub
2. Skin Variables error pops up

### Checklist

- [X] I have made a separate issue for each bug that I am reporting.
- [X] I have searched the issues and this bug has not been reported.
- [X] I am using the most recent skin version.
- [X] I have confirmed that this bug does not occur in the default skin.

**Comment by jurialmunkey** [comment]
full log please. no snippets.

**Comment by jurialmunkey** [comment]
Resolved in SV 2.1.13

**Comment by OfficerKD637** [comment]
This has gotta be a world record! You resolved this quicker than I took to get the log uploaded. Wow!
